### PR TITLE
feat(wallets): add coinshift to allowed domains

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary
Context: https://cowservices.slack.com/archives/C036G0J90BU/p1727685276349879
Allowed using Coinshift with CoW Swap

# Test

- [ ] CoWSwap should work with Safe as before

### CODE REVIEW

**Review by elena-zh — CHANGES_REQUESTED** [review]
Hey @shoom3301 , when I open the app inside the Safe it appears to be not connected 
image
image
Same for the widget
image

Could you please take a look at it?

**Comment by shoom3301** [comment]
@elena-zh fixed

**Comment by shoom3301** [comment]
@elena-zh it is ok, the widget configurator is not suposed to work in Safe Apps

**Review by elena-zh — APPROVED** [review]
Not an issue according to this [discussion ](https://cowservices.slack.com/archives/C0361CDG8GP/p1733238527123859?thread_ts=1733238242.074289&cid=C0361CDG8GP)

**Review by alfetopito — APPROVED** [review]

File: libs/wallet/src/web3-react/connection/safe.tsx [hunk 1]
```diff
@@ -7,7 +7,19 @@ import { Web3ReactConnection } from '../types'
...
   (actions) =>
-    new AsyncConnector(() => import('@web3-react/gnosis-safe').then((m) => new m.GnosisSafe({ actions })), actions)
+    new AsyncConnector(
+      () =>
+        import('@web3-react/gnosis-safe').then(
+          (m) =>
+            new m.GnosisSafe({
+              actions,
+              options: {
+                allowedDomains: [/app\.safe\.global$/, /(.+\.)?coinshift\.global$/, /localhost:5173$/],
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply libs/wallet/src/web3-react/connection/safe.tsx:17]: Yes, in the guide: https://intercom.help/coinshift-d769bc455cc3/en/articles/9779372-a-guide-to-integrate-your-dapp-with-coinshift-personal

**Pull request merged**